Src/GBX.NET/Engines/Game/CGameCtnCollection.chunkl

```diff
@@ -67,28 +67,28 @@ CGameCtnCollection 0x03033000
  ident Vehicle
 
 0x00B
- int
+ int BlocksShadow
 
 0x00C
  bool IsEditable
- int
+ int<ECollectionType> CollectionType
 
-0x00D
+0x00D //IconFid(0xc0), IconSmallFid(0xc4)
 
 0x00E
  int SortIndex
 
 0x011
- int
+ int Default Decoration
```

Comment:
something weird happend here. you dont want spaces in member names. you mean either `DefaultDecoration` or `Default // Decoration`?